rbstor/basic_test.go

```diff
@@ -66,10 +68,10 @@ func TestFullGroup(t *testing.T) {
 	})
 
 	ctx := context.Background()
-
-	workerGate := make(chan struct{}, 1)
-
-	ri, err := Open(td, WithWorkerGate(workerGate))
+	// TODO there is no more worker gate; make this play nice with tests
+	// workerGate := make(chan struct{}, 1)
+	// ri, err := Open(td, WithWorkerGate(workerGate))
```

Comment:
Really I think we want to test the "worker" related things with some intermediate interfaces internally, but imho that's not entirely valuable right now, at least not before we have the core functionality settled